modules/oxide/src/less/theme/content/selection/selection.less

```diff
@@ -122,10 +122,28 @@
 
 .ephox-snooker-resizer-cols {
   cursor: col-resize;
+
+  // Increase the area of the resize handle
+  &::after {
+    content: "";
+    display: block;
+    height: 100%;
+    margin-left: -3px;
```

Comment:
These values don't account for the fact the `after` psuedo selector is placed after the element so it should be `-2px` here instead otherwise it'll be offset 1px to the left of center. eg this is what it currently looks like:

![image](https://user-images.githubusercontent.com/1575550/115168924-f1df9b00-a0ff-11eb-83d6-12741430de40.png)